mm/2s2h/Rando/Types.h

```diff
@@ -2381,6 +2381,11 @@ typedef enum {
     RI_MUSHROOM,
     RI_NONE,
     RI_OCARINA,
+    RI_OCARINA_A_BUTTON,
+    RI_OCARINA_C_DOWN_BUTTON,
+    RI_OCARINA_C_LEFT_BUTTON,
+    RI_OCARINA_C_RIGHT_BUTTON,
+    RI_OCARINA_C_UP_BUTTON,
```

Comment:
Personal preference, I think it's easier to read like so:
```suggestion
    RI_OCARINA_BUTTON_A,
    RI_OCARINA_BUTTON_C_DOWN,
    RI_OCARINA_BUTTON_C_LEFT,
    RI_OCARINA_BUTTON_C_RIGHT,
    RI_OCARINA_BUTTON_C_UP,
```